README.md

```diff
@@ -115,6 +117,7 @@ Use the following properties to configure your instance.
 * **expand** (`false`) - Allows your variables to be "expanded" for reusability within your `.env` file.
 * **defaults** (`false`) - Adds support for `dotenv-defaults`. If set to `true`, uses `./.env.defaults`. If a string, uses that location for a defaults file. Read more at [npm](https://www.npmjs.com/package/dotenv-defaults).
 * **ignoreStub** (`false`) - Override the automatic check whether to stub `process.env`. [Read more here](#user-content-processenv-stubbing--replacing).
+* **prefix** (`'process.env'`) - The prefix to use before the name of your env variables.
```

Comment:
I think I would just include the dot notation here at the end (we pack DefinePlugin is just looking for raw strings, it also keeps opportunities for other notations as a viable option).

“process.env.” as the prefix.

Reply:
The problem with this (original) solution was that it forced an implied assumption that the dot is there. 
Is there any reasonable use case for keeping the trailing dot explicit in the prefix?